app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -432,6 +476,31 @@ const MeetingEditor = () => {
                     )}
                   </ConfirmModal>
                 )}
+                {isEditPage &&
+                  canDelete &&
+                  localStorage.getItem(`meeting-${meeting.id}-report`) !==
+                    null && (
+                    <ConfirmModal
+                      title="Hente inn referat fra localstorage?"
+                      message={`Fant ${
+                        localStorage.getItem(`meeting-${meeting.id}-report`) !==
+                        null
+                          ? 1
+                          : 0
```

Comment:
Don't need to check twice

Reply:
Also, just writing `en` feels more "human" since this will always just be one.

Reply:
Fixed